code/modules/paperwork/paperplane.dm

```diff
@@ -87,7 +87,7 @@
 	add_fingerprint(user)
 
 
-/obj/item/weapon/paperplane/throw_at(atom/target, range, speed, mob/thrower, spin=1, diagonals_first = 0) //prevent the paper plane from spinning
+/obj/item/weapon/paperplane/throw_at(atom/target, range, speed, mob/thrower, spin=0, diagonals_first = 0) //prevent the paper plane from spinning
```

Comment:
This won't do anything, the spin value is being passed in from further up the call chain, most likely wherever the item is actually being thrown from (mob throw code).

Setting it here only sets the *default* value when the spin argument is not specified in higher calls.

The right way to force a specific value to always be one thing, no matter the original calling value

Don't include my comment without shortening it by the way it's just to indicate the code
```

/obj/item/weapon/paperplane/throw_at(atom/target, range, speed, mob/thrower, spin=FALSE, diagonals_first = FALSE)
    //call the parent and explicitly set spin to be false, passing all other values in untouched
    . = ..(target, range, speed, thrower, spin=FALSE, diagonals_first)
```